tests/scripts/thread-cert/node.py

```diff
@@ -1762,13 +1762,17 @@ def ping(self, ipaddr, num_responses=1, size=None, timeout=5):
 
         responders = {}
 
+        received_statistics = False
+
         result = True
         # ncp-sim doesn't print Done
         done = (self.node_type == 'ncp-sim')
-        while len(responders) < num_responses or not done:
+        while len(responders) < num_responses or not done or not received_statistics:
```

Comment:
```suggestion
        while not done or not received_statistics:
```
We can exit as long as `received_statistics` is True because the statistics should be the last line? But we need to set result to False when `len(responders) < num_responses`. 

Thoughts?